### PR TITLE
Configuration memo should return something truthy or this will change

### PR DESCRIPTION
**Pull request by film42**
cc @brianstien @mmmries

### CODE REVIEW

File: lib/action_subscriber/configuration.rb [hunk 1]
```diff
@@ -72,7 +72,7 @@ def self.configure_from_yaml_and_cli(cli_options = {}, reload = false)
             ::ActionSubscriber.config.__send__("#{key}=", setting) if setting
           end
 
-          nil
+          true
```
Comment (film42) [inline lib/action_subscriber/configuration.rb]: I could also return `self` here.

**Comment by mmmries** [comment]
I have no preference around what gets returned. I don't imagine people will depend on the return value of this method. Maybe we should just stick with a literal `true` to make it as simple as possible

**Comment by film42** [comment]
There's a bad spec that needs to be fixed. I already have the seed saved and will fix later. Re-ran travis and specs are passing.

**Comment by mmmries** [comment]
:shipit:

**Pull request merged**